js/src/forum/utils/isFollowingPage.js

```diff
@@ -1,3 +1,7 @@
 import app from 'flarum/forum/app';
 
-export default () => m.route.get().includes(app.route('following'));
+export default () => {
+    if (!app.initializers.has('subscriptions')) return;
```

Comment:
if it's to check that the extension is enabled, this would probably work better
```suggestion
    if (!"flarum-subscriptions" in flarum.extensions) return;
```

Reply:
I think this solution is better. Could also combine the two into one boolean statement?

```js
() => 'flarum-subscriptions' in flarum.extensions && m.route.get().includes(app.route('following'))`
```